app/assets/stylesheets/users.scss

```diff
@@ -1,3 +1,79 @@
 // Place all the styles related to the users controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+
+.login-btn {
+  position: absolute;
+  color: aqua;
+}
+
+.actions {
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin-top: 12px;
+  background-color: transparent;
+  border: none;
+  padding-right: 10px;
+  z-index: 999;
+
+  input {
+    background-color: transparent;
+    border: none;
+    color: #fff;
+  }
+}
+
+.sign-up {
+  height: 100vh;
+  z-index: 1;
+
+  header {
+    padding: 10px 20px;
+    height: 65px;
+
+    h2 {
+      padding-right: 40px;
+    }
+  }
+
+  form {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #fff;
+    border-top: 2px solid #dee2e6;
+    border-bottom: 2px solid #dee2e6;
+
+    .field {
+      //   border-top: 2px solid rgb(212, 212, 212);
```

Comment:
- [ ] Here as well. I would recommend removing the commented code to make the codebase cleaner.